File: stu_spring/webflux/test/auth/src/main/java/com/webflux/auth/domain/user/exception/UserAlreadyExistException.java
package com.webflux.auth.domain.user.exception;

import com.webflux.auth.global.error.exception.ErrorCode;
import com.webflux.auth.global.error.exception.GlobalException;

public class UserAlreadyExistException extends GlobalException {
    public UserAlreadyExistException() {
        super(ErrorCode.USER_ALREADY_EXIST);
    }
}

File: stu_spring/kotlin_basic/src/main/kotlin/com/kotlin/test/kotlintest/domain/user/exceptions/UserAlreadyExistException.kt
package com.kotlin.test.kotlintest.domain.user.exceptions

import com.kotlin.test.kotlintest.global.exception.GlobalException
import com.kotlin.test.kotlintest.global.exception.enums.ErrorCode

class UserAlreadyExistException: GlobalException(ErrorCode.USER_ALREADY_EXIST) {
    companion object {
        @JvmField
        val EXCEPTION = UserAlreadyExistException()
    }
}